text/0000-integer-to-bytes.md

```diff
@@ -0,0 +1,29 @@
+- Start Date: 2014-01-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+The Standard Library currently does not provide a way to cast an integer into a byte array. This RFC would add such methods. 
+
+# Motivation
+
+
+There are some places where rust replicates the behaviour of these functions, and they should probably be replaced by using this. [[1]](https://github.com/rust-lang/rust/blob/master/src/libstd/io/extensions.rs#L80)[[2]](https://github.com/rust-lang/rust/blob/master/src/libstd/io/extensions.rs#L121)[[3]](https://github.com/rust-lang/rust/blob/master/src/librustc_back/sha2.rs#L24) Also a lot of users seem to be asking around how to do this in rust. It is probably a good decision to put this in the standard library. 
+
+# Detailed design
+
+An implementation of the RFC can already be found [here](https://github.com/Binero/rust/blob/master/src/libcore/num/mod.rs#L373), [here](https://github.com/Binero/rust/blob/master/src/libcore/num/mod.rs#L464) and [here](https://github.com/Binero/rust/blob/master/src/libcore/num/mod.rs#L594). 
+It was implemented in libcore. Tests (or examples as you please) can be found [here](https://github.com/Binero/rust/blob/master/src/libcoretest/num/mod.rs#L122).
```

Comment:
You should probably use Github links which include the commit hash, e.g. `https://github.com/Binero/rust/blob/8a39638f02ea5600c0292d82c647ef77d14d525a/src/libcore/num/mod.rs#L373` instead of `https://github.com/Binero/rust/blob/master/src/libcore/num/mod.rs#L373`. You can easily get those by pressing `y` (it will reload the page with the new URL).